realgud-trepan-xpy.el

```diff
@@ -27,7 +27,7 @@
 ;;; Commentary:
 
 ;; realgud support for the Python trepan-xpy
-;; See https://pypi.org/project/trepan-xpy/
+;; See https://pypi.org/project/trepan2/
```

Comment:
The PyPI link is https://pypi.org/project/trepanxpy/

Sorry for the confusion.

Reply:
If you correct the link, I'll merge and update ELPA. Thanks for doing this.

Reply:
Will do!

Reply:
Did and done!